db/migrate/20220823134044_add_ast_digest_index_to_submission_representations.rb

```diff
@@ -0,0 +1,5 @@
+class AddAstDigestIndexToSubmissionRepresentations < ActiveRecord::Migration[7.0]
+  def change
+    add_index :submission_representations, %i[submission_id ast_digest], unique: false, if_not_exists: true
```

Comment:
@iHiD I think this index needs to be on `ast_digest` only.

```suggestion
    add_index :submission_representations, %i[ast_digest], unique: false, if_not_exists: true
```

I don't know if this can be changed or if it needs a new migration to remove and then add this?